packages/graphql/src/schema/resolvers/top-level-union-read.ts

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright (c) "Neo4j"
+ * Neo4j Sweden AB [http://neo4j.com]
+ *
+ * This file is part of Neo4j.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { GraphQLResolveInfo } from "graphql";
+import translateToplevelUnionRead from "../../translate/translate-top-level-union-read";
+import { Context } from "../../types";
+import { execute } from "../../utils";
+import getNeo4jResolveTree from "../../utils/get-neo4j-resolve-tree";
+import { Union } from "../../classes/Union";
+
+export default function topLevelUnionReadResolver({ union }: { union: Union }) {
+    async function resolve(_root: any, args: any, _context: unknown, info: GraphQLResolveInfo) {
+        const context = _context as Context;
```

Comment:
Do we need this casting?

```suggestion
    async function resolve(_root: any, args: any, context: Context, info: GraphQLResolveInfo) {
```